### DISCUSSION TITLE
No Collision between constrained rigidbodies after constraint has broken (Blender)

### DISCUSSION THREAD

**Discussion by scorpion81** [bulletphysics/bullet3#3512] [General]
This potential bug in bullet occurs in latest blender 2.77 buildbot version and has been reported in blenders bugtracker as well, but the feedback (via IRC) was that bullet might  be the cause. Especially a couple of changes in sequential impulse constraint solver between bullet 2.82 and 2.83 could be the culprit. In Blender, bullet was upgraded to 2.83 recently.
Objects still penetrate each other after the constraint has broken IF disableLinkedCollision is disabled.
Works without disableLinkedCollision, but this yields different simulation behavior in general.
So far only the viewport integration of bullet into blender was tested, not the game engine.

See here also https://developer.blender.org/T47758 for more details and a testfile.

**Comment by brechtvl** [comment]
It seems this is a bug in Bullet, introduced in #165 . The previous code would check if the constraint is enabled before disabling collisions, the new code no longer does that as far as I can tell.

**Comment by erwincoumans** [comment]
Thanks for the report Brecht, I'll look into fixing it soon.
